### DISCUSSION TITLE
Vertical Favorites Bar

### DISCUSSION THREAD

**Discussion by N3C2L** [madprops/grasshopper#89] [Ideas]
I'd be glad if the Favorites Bar could be displayed vertically on the left side of the item list.
This way, I would have more space from top to bottom and allowing me to reach my buttons more quickly. Additionally, aligning the Favorites Bar vertically would free up space at the top for another tab to be visible. To enhance the distinction between the vertical Favorites Bar and the item list, it would be wonderful to have the option to set the background color for the vertical Favorites Bar. If the vertical Favorites Bar had reduced margins from the edge and the item list, it would certainly fit seamlessly into the Grasshopper interface.

I would prefer to trigger my favorite filters at the top of the filter button with a left-click (instead of the current right-click). Perhaps it would be possible to implement an option for users to swap mouse buttons specifically for this filter button? This way, a left-click would activate the Favorites Filter, and a right-click would open the current standard filter menu.

**Comment by madprops** [comment]
Check out the new Favorites setting category

**Reply by N3C2L** [reply]
Thank you!

**Reply by madprops** [reply]
I'm gonna upload some tweaks to it soon.

**Comment by N3C2L** [comment]
This idea has been implemented, so I can close this thread.
